scripts/generate_templates_gen3.sh

```diff
@@ -41,16 +41,17 @@ set -e
 
```

Comment:
Do you have an update for the runtime estimate on line 24? (I'd imagine not, if you never ran this script in its current form, but I wanted to check.)

Reply:
No runtime estimate, but it might be good to test this script sometime in the future. I'll make a note of it!